pyresample/geometry.py

```diff
@@ -298,8 +305,29 @@ def get_bbox_lonlats(self) -> tuple:
                            (s4_lon.squeeze(), s4_lat.squeeze())])
         if hasattr(lons[0], 'compute') and da is not None:
             lons, lats = da.compute(lons, lats)
+        if force_clockwise and not self._corner_is_clockwise(
+                lons[0][-2], lats[0][-2], lons[0][-1], lats[0][-1], lons[1][1], lats[1][1]):
+            # going counter-clockwise
+            # swap the side order and the order of the values in each side
+            # to make it clockwise
+            lons = [lon[::-1] for lon in lons[::-1]]
+            lats = [lat[::-1] for lat in lats[::-1]]
         return lons, lats
 
+    @staticmethod
+    def _corner_is_clockwise(lon1, lat1, corner_lon, corner_lat, lon2, lat2):
+        from pyresample.spherical import Arc, SCoordinate
+        point1 = SCoordinate(math.radians(lon1), math.radians(lat1))
+        point2 = SCoordinate(math.radians(corner_lon), math.radians(corner_lat))
+        point3 = SCoordinate(math.radians(lon2), math.radians(lat2))
+        arc1 = Arc(point1, point2)
+        arc2 = Arc(point2, point3)
+        # a straight line has an angle of 0
+        # if the second segment is clockwise the angle will go negative
+        # if the second segment is counter-clockwise the angle will go positive
+        angle = arc1.angle(arc2)
+        return -np.pi < angle < 0
```

Comment:
Just for extra clarity
```suggestion
        is_clockwise = -np.pi < angle < 0
        return is_clockwise
```